### DISCUSSION TITLE
Adding a non tech item, cannot find button id, can someone assist?

### DISCUSSION THREAD

**Discussion by tianrich** [jef/streetmerchant#2604] [Help]
Hey everyone,

Loving the programe, just having an issue with stock showing as available.

I am looking to add Aldi but I cannot seem to show results of in stock items.

Here is what i have so far (first 10 rows of line):

import {Store} from './store';

export const Aldi: Store = {
  currency: '£',
  labels: {
    inStock: {
      container: '#addToBasketButton',
      text: ['inStock'],
    },
  },

The items are showing but always as out of stock, I cannot seem to get the in stock available 
 
[20:43:53] info :: ✖ [aldi] [ambiano (kitchen)] home :: OUT OF STOCK

Can someone help please?

**Comment by Mr-Tech-13** [comment]
Are they in stock on the site?
